### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by heypnus**
Fix some issues and update for the latest changes in 3.1.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,16 +13,19 @@ and it will take care of deploying NSX integration components, and precisely:
...
-In addition, the nsx-container-plugin operator monitors nodes ensuring the
-corresponding NSX logical port is enabled as a container host logical port.
+The nsx-container-plugin operator also monitors the nsx-node-agent status and
+updates the network status on relevant nodes.
+
+In addition, the nsx-container-plugin operator is able to monitor nodes ensuring
+the corresponding NSX logical port is enabled as a container host logical port.
+
+For Openshift 4 clusters, the nsx-container-plugin operator especially monitors
+the network CR in the config.openshift.io namespace to update the container network
```
Comment (dantingl) [inline README.md:27]: It is not `config.openshift.io namespace`, how about Network.config.openshift.io CR?

File: README.md [hunk 2]
```diff
@@ -70,13 +73,16 @@ $ openshift-install --dir=MY_CLUSTER create ignition-configs
 This bootstrap ignition file will be added to the terraform tfvars.
 Then use terraform to install Openshift 4 cluster on vSphere.
 
+### Upgrade
+
+For upgrading, all yaml files in `deploy/${platform}/` should be involved.
```
Comment (dantingl) [inline README.md:78]: need to update image as well

File: README.md (hunk 1 above)
Comment (heypnus) [reply README.md:27]: Good eyes, it's a historical problem... Fixed, thanks!

File: README.md (hunk 2 above)
Comment (heypnus) [reply README.md:78]: done

**Review by dantingl — APPROVED** [review]

**Pull request merged**